gen_systemd_service_file.py

```diff
@@ -4,13 +4,18 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 
-import package.systemd_files_generator as gen
+from docker.package.packages import *
+from docker.package.model import *
 import sys
 
 if len(sys.argv) > 1:
-    service_name = sys.argv[1]
-    if len(sys.argv) == 3:
-        service_file = gen.gen_service(service_name, sys.argv[2])
-    else:
-        service_file = gen.gen_service(service_name)
-    gen.print_service_file(service_file, f"{service_name}.service")
+    binary_name = sys.argv[1]
+    for package in packages.packages:
+        if binary_name == package.name:
+            for systemd_unit in package.systemd_units:
+                if systemd_unit.suffix is None:
+                    print_service_file(systemd_unit.service_file,
+                                       f"{package.name}.service")
+                else:
+                    print_service_file(systemd_unit.service_file,
+                                       f"{package.name}-{systemd_unit.suffix}.service")
```

Comment:
Probably it would be a bit shorter if you defined `suffix = None if systemd_unit.suffix is None else f"-{systemd_unit.suffix}"` and then typed `print_service_file` just once.